### PR TITLE
post-war spaceport news: alien paranoia and Deneb researchers

### PR DESCRIPTION
**Pull request by NomadicVolcano**
This adds an event that generates spaceport news about alien paranoia, plus some scientific spaceport news in the Deneb system. The event is triggered by the completion of the main story.

This is already included in #4952, but it is good enough to stand on its own, so I'm doing a separate PR for it.

### CODE REVIEW

**Comment by NomadicVolcano** [comment]
news-shot-hostile

news-shot-martini

news-shot-food

news-shot-tarazed

news-shot-hologram

news-shot-anarchy

**Comment by NomadicVolcano** [comment]
Researchers in the Deneb system:

news-shot-pug

news-shot-quarg

File: data/human/post-war news.txt [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+	phrase
+		"media name"
+	word
+		"'"
+	word
+		" but now I'm sure that"
+		" but I think"
+		" and I know now that"
+	word
+		" the one of the actors is"
```
Comment (jvpoz) [inline data/human/post-war news.txt:549]:
```suggestion
		" one of the actors is"
```

**Comment by sem-geologist** [comment]
I would consult this "xenoarcheologist's" claim with some physic. I doubt it.

**Comment by MCOfficer** [comment]
> I would consult this "xenoarcheologist's" claim with some physic. I doubt it.

Doesn't have to be right, only has to be in-character.

File: data/human/post-war news.txt [hunk 2]
```diff
@@ -0,0 +1,647 @@
...
+		" in a stylin' party-hall"
+		""
+	word
+		", but after what I heard"
+		", but after what I've seen"
+		", and now that I've really looked at them"
+	word
+		" I'm sure they're"
+		" I know they're"
+		" it's clear to me now they're"
```
Comment (W1zrad) [inline data/human/post-war news.txt:522]:
```suggestion
		" it's clear to me that they're"
```
`, and now that I've really looked at them it's clear to me now they're`: don't like the possibility for the two `now`s close to each other.

File: data/human/post-war news.txt [hunk 3]
```diff
@@ -0,0 +1,647 @@
...
+
+phrase "(reason why someone is an alien)"
+	word
+		"someone saw them"
+		"I saw them"
+	word
+		" sneeze non-stop for an hour"
+		" wearing dark sunglasses in a movie theater"
+		" eating a piece of metal"
+		" speaking in their dreams"
```
Comment (W1zrad) [inline data/human/post-war news.txt:583]: Can't exactly imagine why someone would see most of these people asleep, but it's funny to think about so I say leave it in.

**Comment by NomadicVolcano** [comment]
Closing all pull requests related to this account.

**Comment by Pointedstick** [comment]
What happened here? I was about to review it lol.

**Comment by FixItYondu** [comment]
@Pointedstick His last message before he closed it says that he's closing all PRs related to his account. I assume he's leaving the community.

**Comment by Zitchas** [comment]
We don't know. All we know is that NomadicVolcano closed all their issues and PRs to both main and high-dpi repositories. When I asked what was going on in the Korath Exiles Expansion PR, they posted a link to a google drive folder with all the assets and said that it was a henceforth unmonitored account, so it'd probably disappear in 9 months or so. Other than that, I don't know anything.

**Comment by Pointedstick** [comment]
Went into the witness protection program, maybe. Perhaps someone else should take it over?

**Comment by W1zrad** [comment]
Seeing as this is the place with the most recent conversation post-closing, I think this is the best spot to put this comment. It looks like there were four unfinished content PRs that were not included in MasterofGrey's Hai Campaign PR. One of those is the Korath Exiles Expansion (#4987), which Zitchas is currently in charge of, which leaves three: #4899, Acorn Cookies, #4770, the Alexandria side-story, and #4901, Terraform Warfeed. If nobody's already working on them, I'm interested in taking these, fixing their pending issues, and trying to get them back into a PR-able state. There are a couple concerns I have, however, so I'm looking for some feedback before I start.
First, are these PRs wanted? I'd would assume so, considering they all have solid writing on first glance, but I'd like to confirm that they are before I start spending time on them. On a related note, from what I can tell in the PR conversations, NomadicVolcano was pretty adamant in not making changes to the story they'd laid out. Would you prefer I stay as close to these stories as possible, or try to incorporate some of the story changes others requested that NomadicVolcano denied?
Second, both #4770 and #4901 are partly dependent on the merging of #4764. Seeing as I am, by all definitions, code-illiterate, I wouldn't be able to get #4764 merged on my own. There are a few workarounds I can think of: a) I could restructure the stories to eliminate their dependency on #4764 (though this approach may conflict with trying to stay as faithful to the original as possible), b) I could learn just enough to figure out how to download the files and re-upload them into a new PR, and just take all the suggestions I get (though this may be a convoluted approach), or c) someone else who knows how to code could take on #4764 and re-PR it themselves (though I don't want to tack on work to someone else).
To those reading, I'd really appreciate any opinions, feedback, and suggestions about some of the options I've listed above. I'd actually prefer to wait for others' thoughts as a sort of go-ahead before I get started. Once I have that feedback, I'll comment on all three closed PRs so others know the work's being reuploaded. Let me know what you think!

**Comment by Zitchas** [comment]
I haven't thoroughly read all of them, but I really liked #4899 . While I agree with Amazinite's comment regarding the ending being a tad abrupt, I found it to be a fun story and an amusing look at Hai culture. Definitely a mission set I would like to see in-game.

To be honest, the story highlights a problem I've seen with the way the Hai/Unfettered relationship is portrayed. The Hai keep portraying it as just a minor thing. To hear them talk about it, they're not at war, they just have some rowdy neighbours. But then the chokpoint system is basically a war zone where up to a thousand Hai/Unfettered are losing their lives anytime we stay there for more than a few minutes. If that's "just a casual dispute with a rowdy neighbour", then my opinion of the Hai just shifted to considering them a very violent and warring species, despite the utopic impression I get from the rest of their civilization. (although to be fair, they have a lot of hints and planet names and stuff that seem to hint at a really dark undercurrent). I'm not sure if Acorn Cookies is the best place to resolve this problem, but it is worth thinking about, anyway.

**Comment by Amazinite** [comment]
I reviewed Acorn Cookies, and I'm interested in reworking it myself to get it merged. I have no comment on Warfeed and Alexandria though as I hadn't looked at those prior to them being closed. As far as the those two requiring #4764 goes, I'm dubious as to if that's actually the case. The issue of #1938 is really just a visual bug, so I don't see how it could actually impact the mission structure of a string.

**Comment by W1zrad** [comment]
Thanks for the response, and thanks Amazinite for checking the concern relating to #4764. Based on what I've read of the conversation on #4770, it seems like the situation there is similar. Since nobody's been strictly opposed to it, and considering that the decision of whether content is wanted is (in part) made after a PR is submitted, I'll be moving forward with restoring #4770 and #4901. I intend to decide where to make edits to NomadicVolcano's story at my own discretion based on comments left on the original PRs. Thanks for the input, and if you have any other thoughts, please let me know.

**Pull request closed without merging**